gcexport.py

```diff
@@ -197,22 +198,37 @@ def sanitize_filename(name, max_length=0):
 
 
 def write_to_file(filename, content, mode, file_time=None):
```

Comment:
IMHO its good idea to make a default value here. Especially as by reading the docstrings user cant be sure which one to pick.

```suggestion
def write_to_file(filename, content, mode='w', file_time=None):
```